components/style/values/computed/box.rs

```diff
@@ -83,3 +83,59 @@ impl ToComputedValue for SpecifiedFloat {
         }
     }
 }
+
+#[allow(missing_docs)]
```

Comment:
How about having the `spec url` (https://drafts.csswg.org/css-box/#propdef-clear) for `enum Clear` and removing this `#[allow(missing_docs)]`?